src/pages/Profile.jsx

```diff
@@ -0,0 +1,39 @@
+// ApplicationForm.js
+import React from 'react';
+// import { Link }  from 'react-router-dom';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+// Signup.js
+import Signin from '../components/Signin';
```

Comment:
Could you please confirm if the Profile page include the Signin feature? If it doesn't, feel free to remove this import to streamline the code. Thank you!